config/config.go

```diff
@@ -110,6 +115,10 @@ func FromFlags() (*Config, error) {
 	flag.IntVar(&streamTimeout, "stream-timeout", 3, "Defines the timeout in seconds the server waits for the event to be sent to the client")
 	flag.Uint64Var(&forceStartHeight, "force-start-height", 0, "Force set starting Cadence height. This should only be used locally or for testing, never in production.")
 	flag.StringVar(&filterExpiry, "filter-expiry", "5m", "Filter defines the time it takes for an idle filter to expire")
+	flag.StringVar(&cloudKMSKeys, "coa-cloud-kms-keys", "", "COA Cloud KMS keys")
```

Comment:
explain what is the format of the keys provided in the description

Reply:
Added explanation in https://github.com/onflow/flow-evm-gateway/pull/282/commits/f9303971f9c0a580a52904b3cd8e33edef785b53